lib/middleware/add-weather.js

```diff
@@ -0,0 +1,23 @@
+const getForecast = require('../services/weather-api');
+const getLocation = require('../services/maps-api');
+
+module.exports = () => (req, res, next) => {
+  const { address } = req.body;
+
+  getLocation(address)
```

Comment:
get location should not be called inside add-weather. getLocation adds the location to the req.body, so what you should do on line 5 is ```const { location } = req.body```